cloudify_gcp/compute/instance.py

```diff
@@ -379,11 +380,26 @@ def create(instance_type,
             zone = props['zone'] = utils.get_gcp_resource_name(
                     gcp_config['zone'])
 
+    disks = [
+            disk.target.instance.runtime_properties[constants.DISK]
+            for disk
+            in utils.get_relationships(
+                ctx,
+                filter_resource_types='compute#disk'
+                )
+            ]
+    disks.sort(key=lambda disk: disk['boot'], reverse=True)
```

Comment:
The boot disk also has to be the first disk in the list*, so this does that too. I'll add a comment as I had to go and test it to check that was actually true...

*```{
 "error": {
  "errors": [
   {
    "domain": "global",
    "reason": "invalid",
    "message": "Invalid value for field 'resource.disks[0]': ''. Boot disk must be the first disk attached to the instance."
   }
  ],
  "code": 400,
  "message": "Invalid value for field 'resource.disks[0]': ''. Boot disk must be the first disk attached to the instance."
 }
}```